lua/lscs/autorun/lscs_forcepowers.lua

```diff
@@ -132,7 +132,8 @@ if SERVER then
 		if not IsValid( ply ) then return end
 		if not ply:Alive() then return end
 
-		if ply:InVehicle() and not ply:GetAllowWeaponsInVehicle() then return end
+		local AllowForce = hook.Run( "LSCS:PlayerCanForce", ply, nil )
```

Comment:
maybe two different hooks? idk, sometimes passing nil, sometimes not is a bit confusing

Reply:
Could just skip the checks that pass nil altogether.